### ISSUE TITLE
Pressing space bar multiple times in Fennec only creates more spaces

### ISSUE THREAD

**Issue by myfirstnameispaul** [dessalines/thumb-key#685]
### Steps to reproduce

1. In Fennec, press space bar multiple times within a text field on a web page.

### Expected behavior

Auto print characters (e.g., `,`, `.`, `!`, etc.)

### Actual behavior

Spaces only (`  `) are printed.

### Android version

Android 14

### Device

Google Pixel ~~6~~ 5

### Other details

GrapheneOS with Thumb-Key installed from F-Droid.

The issue began some time in the past few weeks. It is not immediately present, but after some time the issue begins and does not stop until I reboot the phone. Even closing all tabs and closing Fennec does not fix the problem.

### Acknowledgements

- [X] I have searched the existing issues and this is a new one, **NOT** a duplicate or related to another open issue.
- [X] This is not a question or a discussion, in which case I should have gone to [lemmy.ml/c/thumbkey](https://lemmy.ml/c/thumbkey)
- [X] This is a **single** bug report,  in case of multiple bugs I will open a separate issue for each one  (they can always link to each other if related)
- [X] I have written a short but informative title.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements. 🤡
- [X] I have filled out all of the requested information in this form.

**Comment by dessalines** [comment]
Does this only happen with Fennec? What about other apps?

**Comment by bdtoole** [comment]
I actually see this behavior in both Google messages and within the "Test out Thumb-Key" section of the app. I restarted my phone and it briefly (for maybe one cycle through each punctuation mark) worked as expected, but then immediately reverted to just adding in spaces.

**Comment by dessalines** [comment]
What version of thumb-key?

**Comment by bdtoole** [comment]
I'm on version 2.6.9 running on a OnePlus 9 Pro

**Comment by agregson** [comment]
I have been experiencing the same issue on Pixel 6 Pro, Graphene OS, Thumb Key 2.6.9 from F-Droid. The behaviour is the same in ALL applications. Rebooting does not help. Thanks

**Comment by dessalines** [comment]
We are on `2.6.15` now.

**Comment by agregson** [comment]
Oops. F-Droid thinks I'm up to date.

**Comment by myfirstnameispaul** [comment]
2.6.9 is the latest in the F-Droid repo.

https://f-droid.org/en/packages/com.dessalines.thumbkey/

**Comment by myfirstnameispaul** [comment]
I know very little about submitting to F-Droid, but it seems there is some sort of problem here?

https://gitlab.com/fdroid/fdroiddata/-/commits/master/metadata/com.dessalines.thumbkey.yml

**Comment by agregson** [comment]
Right. F-Droid only offers up to 2.6.9. 

As far as the space bar punctuation bug, I had to revert back to 2.6.6 to get it to work again. I updated to the latest from Github and it was still not working, even after uninstallation and several reboots. At least 2.6.6 works and provides all the functionality that I require at this time. 

It is a great keyboard, just need more practice! Coming from a Blackberry. Thanks so much for the project!
